package-system/zlib/build_config.json

```diff
@@ -58,7 +58,8 @@
                 "custom_install_cmd": [
                     "./install_zlib_linux.sh"
                 ]
-            }
+            },
+	    "Linux-aarch64": "@Linux"
```

Comment:
Is this case insenstive?
I can authors trying to use `@linux` as well

Reply:
Its case sensitive currently. To make it case-insensitive, we would have to update all 3P libraries existing json files.